source/Inotify.cpp

```diff
@@ -189,7 +189,7 @@ boost::optional<FileSystemEvent> Inotify::getNextEvent()
     // Read Events from fd into buffer
     while (mEventQueue.empty()) {
         length = 0;
-        memset(&buffer, 0, EVENT_BUF_LEN);
```

Comment:
[here is a post with a similar problem](https://cboard.cprogramming.com/c-programming/119201-initializing-character-arrays-memset.html)